contents/blog/mykonos-hackathon.md

```diff
@@ -0,0 +1,153 @@
+---
+date: 2024-05-28
+title: What we built at our Mykonos hackathon
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - james-temperton
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/aruba/beach-hog.jpg
```

Comment:
Feels like we could do something more for the header image?

Doesn't look like we managed to get a photo of the whole group, but here are a couple that could work:

- [Hackathon idea planning](https://photos.google.com/share/AF1QipPHfMFszLUAYjAuRKpPekbr8dVfAB4xpEQ5E0Z_jU5B5aV9qyQStPnW2Q__jbTsxg/photo/AF1QipMNwzk4dXlxlISHWlZbEL_FTEpvHoCzdLS1jSNv?key=NkxyaldxdlFpeTZPNGh4Y2MwcGhuLWJpM1lkOTlB) (topical!)
- [Photo on bus](https://photos.google.com/share/AF1QipPHfMFszLUAYjAuRKpPekbr8dVfAB4xpEQ5E0Z_jU5B5aV9qyQStPnW2Q__jbTsxg/photo/AF1QipNzKBRJb0DhVPdueS9Lxzgq1SeuqdrGHek-K4se?key=NkxyaldxdlFpeTZPNGh4Y2MwcGhuLWJpM1lkOTlB)

Reply:
+1 to this - I prefer the Lottie art personally for headers, but since we already have the Greek art from the stickers @lottiecoxon did we could easily use those here